pkgs/os-specific/linux/lxc/default.nix

```diff
@@ -58,6 +58,7 @@ stdenv.mkDerivation rec {
     "localstatedir=\${TMPDIR}"
     "sysconfdir=\${out}/etc"
     "sysconfigdir=\${out}/etc/default"
+    "bashcompdir=\${out}/etc/bash_completion.d"
```

Comment:
Please put bash completion files into `$out//share/bash-completion/completions`, *not* into `$out/etc`.